packages/analytics-js/src/state/slices/storage.ts

```diff
@@ -1,10 +1,53 @@
 import { signal } from '@preact/signals-core';
-import { StorageState } from '@rudderstack/analytics-js-common/types/ApplicationState';
+import {
+  UserSessionEntries,
+  StorageState,
+} from '@rudderstack/analytics-js-common/types/ApplicationState';
+import { DEFAULT_STORAGE_TYPE } from '@rudderstack/analytics-js-common/types/Storage';
+import { userSessionStorageKeys } from '@rudderstack/analytics-js/components/userSessionManager/userSessionStorageKeys';
+
+const defaultEntries: UserSessionEntries = {
```

Comment:
This elaborate default value object is not required. All we need is a list of entry names which is already an enum.
I'm aware how you are using it in StoreManager but that should be changed.